src/locale/locales/fr/messages.po

```diff
@@ -4499,47 +4306,43 @@ msgstr "Avertir"
 
 #: src/screens/Onboarding/StepAlgoFeeds/index.tsx:134
 msgid "We also think you'll like \"For You\" by Skygaze:"
-msgstr ""
+msgstr "Nous pensons également que vous aimerez « For You » de Skygaze :"
 
 #: src/screens/Deactivated.tsx:133
 msgid "We estimate {estimatedTime} until your account is ready."
-msgstr ""
+msgstr "Nous estimons à {estimatedTime} pour que votre compte soit prêt."
```

Comment:
You have to reverse the sentence so it works (without that, it would miss a `le temps nécessaire` after `{estimateTime}` which would make the sentence too long imho). 

```suggestion
msgstr "Nous estimons que votre compte sera prêt dans {estimatedTime}."
```